test/test_historical_locations.js

```diff
@@ -1,5 +1,6 @@
 import commonTests from './common';
+import { historicalLocations } from './constants';
 
 const mandatory     = ['time'];
 
-commonTests('HistoricalLocations', mandatory);
+commonTests(historicalLocations, 8882, mandatory);
```

Comment:
Ports are not released until all the suites are completed. So we need to use one per test suite.